lib/ts/recipe/passwordless/api/implementation.ts

```diff
@@ -248,6 +249,24 @@ export default function getAPIImplementation(): APIInterface {
                 session: input.session,
             });
 
+            const anomalyDetection = AnomalyDetectionRecipe.getInstance();
+            if (anomalyDetection !== undefined) {
+                const anomalyDetectionResult = await anomalyDetection.recipeInterfaceImpl.checkAnomalyWithRequest({
+                    request: input.options.req,
+                    status: postAuthChecks.status,
+                    userId: (response.user ?? authenticatingUser!.user).id,
+                    tenantId: input.tenantId,
+                    action: isSignUp ? "SIGN_UP" : "SIGN_IN",
```

Comment:
so for the case of sign up, this is happening after you have already created a user above (consumeCode creates a user). So it doesn't really make sense to put it here (after sign up)

Reply:
if we return the status SIGN_IN_UP_NOT_ALLOWED are the end-user still be able to sign-in?

Reply:
yes cause the session tokens are attached to the response + the user is created in the supertokens db anyway, which i assume we want to avoid.

Reply:
ok